notebooks/tel_and_site/subsys_req_ver/m2/SITCOM-1487_TEA_vibration_analysis_MTM2.ipynb

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
<p>So, Laura gave me this feedback and I am passing it to you! Perhaps you could write a markdown block with instructions that we have to uncomment the time windows of interest that we want the EFD data from </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>OK, done!</p>